node/taskfiles/taskfile.local.yml

```diff
@@ -102,10 +102,10 @@ tasks:
     dotenv: [".env"]
     cmds:
       - go test ./pkg/libp2p/tests -v
-  # test-reporter:
-  #   dotenv: [".env"]
-  #   cmds:
-  #     - go test ./pkg/reporter -v
+  test-reporter:
+    dotenv: [".env"]
+    cmds:
+      - go test ./pkg/reporter -v -count=1
```

Comment:
why is count=1 required?

Reply:
ah that's for local (to tell go not to cache the test results), will remove it